test/controllers/pages_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class PagesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```

Comment:
Maybe it would be good to add a test in here that tests we get a 200 response when requesting the root. Also, if you'd like to learn more about the testing framework used within Portal, then have a look at some of the documentation on [RSpec](https://relishapp.com/rspec/rspec-rails/v/3-9/docs).

Reply:
This has been a very helpful resource. If you have any additional resources you have found useful when learning rspec, please let me know! Also, if there are any sections of portal which you believe have particularly nice testing to model off, I'd be very appreciative to hear your opinion and take a look.

Reply:
I would say I'm still learning RSpec myself really. I will have a look and see if I can find any good resources. Within Portal I'm not sure if there are any fantastic examples so I'll just send you to one that we've created as a team, so then I can at least answer questions about it. So if you look at the [seamless_login_access_controller_spec.rb](https://github.com/wpengine/my_wpe/blob/master/spec/controllers/seamless_login_access_controller_spec.rb) file it contains a reasonable set of examples using RSpec.

Reply:
This is useful, thank you.